oktoberfest/data/spectra.py

```diff
@@ -286,6 +282,23 @@ def from_hdf5(cls: Type[SpectraT], input_file: Union[str, Path]) -> SpectraT:
         """
         return cls(anndata.read_h5ad(str(input_file)))
 
+    def remove_decoys(self) -> None:
+        """Remove decoys in-place."""
+        self.__dict__ = Spectra(self[~self.obs.REVERSE].copy()).__dict__
+
+    def filter_by_score(self, threshold: float) -> None:
+        """Filter out peptides with Andromeda score below threshold in-place."""
+        self.__dict__ = Spectra(self[self.obs.SCORE >= threshold].copy()).__dict__
+
+    def remove_duplicates(self, num_duplicates: int) -> None:
+        """Filter out (peptide, charge, collision energy) duplicates if there's more than n_duplicates."""
+        # TODO
+        pass
+
+    def standardize_fragmentation_names(self) -> None:
+        """Replace non-abbreviated fragmentation method spellings in-place."""
+        self.obs.replace({"FRAGMENTATION": {"electron transfer dissociation": "ETD"}}, inplace=True)
```

Comment:
please check if this cannot be done using the fragmentationMethod param in the config. The idea is to put more and more of the metadata stuff into the config, so that the users are mor flexible.

Reply:
Don't have the data to reproduce the original issue where spelled-out fragmentation method names ended up in the refinement dataset anymore - however, this should cause problems much sooner as `annotate_spectra` and `retrieve_ion_types` from spectrum_fundamentals won't work if the fragmentation method names aren't standardized, so this is the wrong place to handle it anyways. Removed this method.